src/components/PositionsList/PositionsList.stories.tsx

```diff
@@ -1,91 +1,126 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { storiesOf } from '@storybook/react'
-import { PositionsList } from './PositionsList'
+import { ILiquidityItem, PositionsList } from './PositionsList'
 import { Grid } from '@material-ui/core'
 import { MemoryRouter } from 'react-router-dom'
+import { PaginationList } from './Pagination/Pagination'
+
+const data = [
+  {
+    tokenXName: 'BTC',
+    tokenYName: 'SNY',
+    tokenXIcon:
+      'https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v/logo.png',
+    tokenYIcon:
+      'https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v/logo.png',
+    min: 2149.6,
+    max: 149.6,
+    fee: 0.05,
+    tokenXLiq: 5000,
+    tokenYLiq: 300.2,
+    value: 10000.45,
+    id: '1'
+  },
+  {
+    tokenXName: 'BTC',
+    tokenYName: 'SNY',
+    tokenXIcon:
+      'https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v/logo.png',
+    tokenYIcon:
+      'https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v/logo.png',
+    min: 2149.6,
+    max: 149.6,
+    fee: 0.05,
+    tokenXLiq: 5000,
+    tokenYLiq: 300.2,
+    value: 10000.45,
+    id: '2'
+  },
+  {
+    tokenXName: 'BTC',
+    tokenYName: 'SNY',
+    tokenXIcon:
+      'https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v/logo.png',
+    tokenYIcon:
+      'https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v/logo.png',
+    min: 2149.6,
+    max: 149.6,
+    fee: 0.05,
+    tokenXLiq: 5000,
+    tokenYLiq: 300.2,
+    value: 10000.45,
+    id: '3'
+  },
+  {
+    tokenXName: 'BTC',
+    tokenYName: 'SNY',
+    tokenXIcon:
+      'https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v/logo.png',
+    tokenYIcon:
+      'https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v/logo.png',
+    min: 2149.6,
+    max: 149.6,
+    fee: 0.05,
+    tokenXLiq: 5000,
+    tokenYLiq: 300.2,
+    value: 10000.45,
+    id: '4'
+  }
+]
+
+function paginator(data: ILiquidityItem[], currentPage: number, perPageItems: number) {
```

Comment:
pagination is already inside component, why is it also once again inside its story?